src/components/ComputeModal.js

```diff
@@ -1121,10 +1129,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           ])],
           [willDeletePersistentDisk(), () => h(Fragment, [
             p([
-              'Reducing the size of a persistent disk requires it to be deleted and recreated. This will ',
+              'To reduce the size of the PD, the existing PD will be deleted and a new one will be created and attached to your virtual machine instance. This will ',
               span({ style: { fontWeight: 600 } }, ['delete all files on the disk.'])
             ]),
-            h(SaveFilesHelp)
+            isToolRStudio() ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp) 
```

Comment:
With the above, it may be nice to prevent the need for a lookup, as in:
```suggestion
            getExistingEnvironmentConfig().runtime.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp) 
```
That way, you don't have to wonder where `isToolRStudio` is obtaining its information.